test/system/160-volumes.bats

```diff
@@ -125,13 +126,17 @@ EOF
         expect_msg='.* exec user process caused.*permission denied'
     fi
 
-    run_podman ${expect_rc} run --rm --volume $myvolume:/vol:z $IMAGE /vol/myscript
+    run_podman ${expect_rc} run --rm --volume $myvolume:/vol:noexecz $IMAGE /vol/myscript
```

Comment:
"noexecz"? Should there be a comma or something?

Reply:
This is what happens when I push before the tests finish running. Oops.